govuk_frontend_jinja/templates/template.html

```diff
@@ -27,7 +27,7 @@
     {# image url needs to be absolute e.g. http://wwww.domain.com/.govuk_frontend_jinja/.govuk_frontend_jinja/govuk-opengraph-image.png #}
     <meta property="og:image" content="{{ assetUrl | default('/assets') }}/images/govuk-opengraph-image.png">
   </head>
-  <body class="govuk-template__body {{ bodyClasses }}" {%- for attribute, value in bodyAttributes %} {{attribute}}="{{value}}"{% endfor %}>
+  <body class="govuk-template__body {{ bodyClasses }}" {%- for attribute, value in (bodyAttributes.items() if bodyAttributes else {}.items()) %} {{attribute}}="{{value}}"{% endfor %}>
```

Comment:
@matthew-shaw This is the same pattern we've used elsewhere. I don't super love it so maybe we can see if there's anything a little terser in future. But this is consistent at least with the other templates.